### ISSUE TITLE
What to change in order gtbot to work?

### ISSUE THREAD

**Issue by atomiclucky** [GrowtopiaNoobs/Growtopia-Bot#10]
Does anyone know what you need to change? I already changed the version, didn't work.

**Comment by likeyrohax** [comment]
change your brains

**Comment by atomiclucky** [comment]
Bruh that's so helpful

**Comment by Greanex** [comment]
Hey bro, where did you changed the version? Please, I need some help.

**Comment by atomiclucky** [comment]
in source files bruh

**Comment by Greanex** [comment]
Ok. But which source file? Can you explain me exactly?

**Comment by Greanex** [comment]
Also what version of Qt you're using?

**Comment by atomiclucky** [comment]
yo help me out, i changed the version string but in debug it shows that it's still outdated. Do you know what string to put in?
![paveikslas](https://user-images.githubusercontent.com/49339987/62878018-f18adc00-bd30-11e9-89cc-3cfe11649733.png)
![paveikslas](https://user-images.githubusercontent.com/49339987/62878096-15e6b880-bd31-11e9-8929-36fe62936916.png)

**Comment by atomiclucky** [comment]
@likeyrohax you know the answer

**Comment by Greanex** [comment]
@atomiclucky Hey, what version of Qt Creator you're using?

**Comment by atomiclucky** [comment]
Bruh do you even read the wiki

**Comment by Greanex** [comment]
@atomiclucky Wiki? Do u mean readme.md?

**Comment by atomiclucky** [comment]
@Greanex its the same n00b

**Comment by Greanex** [comment]
@atomiclucky Hello dude. I installed the recommended version of Qt Creator. But it giving a compiler error when I try to build project. What should I do?

"Qt Creator needs a compiler set up to build. Configure a compiler in the kit options."

**Comment by atomiclucky** [comment]
bruh

**Comment by atomiclucky** [comment]
download vs

**Comment by Greanex** [comment]
Thanks but which version should I download? vs 2013?

**Comment by atomiclucky** [comment]
yes

**Comment by Greanex** [comment]
@atomiclucky Thanks for helping. It worked. But, it giving a compiler error when I build project or run. Here's a ss:
![Ekran Alıntısı](https://user-images.githubusercontent.com/52894087/63114798-657dec00-bf9e-11e9-8e71-2289eded63e6.PNG)

**Comment by GrowtopiaNoobs** [comment]
@atomiclucky Sorry, but I haven't used my bot for while, so I wont be able to help you with it

@Bykosw Please don't be toxic, else you can be banned from github

@Greanex I would recommend you to try using Debug build, if that won't work, then try to see if there isn't somewhere shown at which line error happen and screenshot those pieces of code and send them here. Also you can try all of those options https://stackoverflow.com/questions/7076494/fatal-error-c1001-an-internal-error-has-occurred-in-the-compiler

**Comment by Joiubaxas** [comment]
@GrowtopiaNoobs do you have a discord or any way to contact you? I've been trying to work with growtopia packets myself, but they seem to be obfuscated and I didn't get further than creating a udp/tcp two way proxy to see what packets are being sent.

**Comment by GrowtopiaNoobs** [comment]
They aren't obsfucated, but they are using range coder from enet, so you won't be able to read their content directly. There is some enet plugin for wireshark, but idk if it decodes range encoding.
Discord: Anybody#8639

**Comment by Joiubaxas** [comment]
> They aren't obsfucated, but they are using range coder from enet, so you won't be able to read their content directly. There is some enet plugin for wireshark, but idk if it decodes range encoding.
> Discord: Anybody#8639

I've messaged you on discord.

**Comment by Greanex** [comment]
> They aren't obsfucated, but they are using range coder from enet, so you won't be able to read their content directly. There is some enet plugin for wireshark, but idk if it decodes range encoding.
> Discord: Anybody#8639

Hey, thanks for answering but it giving different error this time:
`22:43:03: Running steps for project Growtopia_Bot...
22:43:03: Configuration unchanged, skipping qmake step.
22:43:03: Starting: "C:\Qt\Qt5.3.2\Tools\QtCreator\bin\jom.exe" 
	C:\Qt\Qt5.3.2\Tools\QtCreator\bin\jom.exe -f Makefile.Release
	cl -c -nologo -Zm200 -Zc:wchar_t -FS -O2 -MD -GR -W3 -w34100 -w34189 -EHsc -DUNICODE -DWIN32 -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_OPENGL_ES_2 -DQT_OPENGL_ES_2_ANGLE -DNDEBUG -I"..\Growtopia_Bot\include" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtWidgets" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtGui" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtANGLE" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtCore" -I"release" -I"." -I"." -I"C:\Qt\Qt5.3.2\5.3\msvc2013\mkspecs\win32-msvc2013" -Forelease\ @C:\Users\mustafa\AppData\Local\Temp\main.obj.7108.141.jom
	cl -c -nologo -Zm200 -Zc:wchar_t -FS -O2 -MD -GR -W3 -w34100 -w34189 -EHsc -DUNICODE -DWIN32 -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_OPENGL_ES_2 -DQT_OPENGL_ES_2_ANGLE -DNDEBUG -I"..\Growtopia_Bot\include" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtWidgets" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtGui" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtANGLE" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtCore" -I"release" -I"." -I"." -I"C:\Qt\Qt5.3.2\5.3\msvc2013\mkspecs\win32-msvc2013" -Forelease\ @C:\Users\mustafa\AppData\Local\Temp\moc_addbots.obj.7108.610.jom
'cl' is not recognized as an internal or external command,
operable program or batch file.
'cl' is not recognized as an internal or external command,
operable program or batch file.
jom: C:\Users\mustafa\Desktop\Growtopia-Bot-master\build-Growtopia_Bot-Desktop_Qt_5_3_MSVC2013_32bit-Release\Makefile.Release [release\main.obj] Error 1
	cl -c -nologo -Zm200 -Zc:wchar_t -FS -O2 -MD -GR -W3 -w34100 -w34189 -EHsc -DUNICODE -DWIN32 -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_OPENGL_ES_2 -DQT_OPENGL_ES_2_ANGLE -DNDEBUG -I"..\Growtopia_Bot\include" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtWidgets" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtGui" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtANGLE" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtCore" -I"release" -I"." -I"." -I"C:\Qt\Qt5.3.2\5.3\msvc2013\mkspecs\win32-msvc2013" -Forelease\ @C:\Users\mustafa\AppData\Local\Temp\mainwindow.obj.7108.313.jom
	cl -c -nologo -Zm200 -Zc:wchar_t -FS -O2 -MD -GR -W3 -w34100 -w34189 -EHsc -DUNICODE -DWIN32 -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_OPENGL_ES_2 -DQT_OPENGL_ES_2_ANGLE -DNDEBUG -I"..\Growtopia_Bot\include" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtWidgets" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtGui" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtANGLE" -I"C:\Qt\Qt5.3.2\5.3\msvc2013\include\QtCore" -I"release" -I"." -I"." -I"C:\Qt\Qt5.3.2\5.3\msvc2013\mkspecs\win32-msvc2013" -Forelease\ @C:\Users\mustafa\AppData\Local\Temp\moc_mainwindow.obj.7108.453.jom
'cl' is not recognized as an internal or external command,
operable program or batch file.
jom: C:\Users\mustafa\Desktop\Growtopia-Bot-master\build-Growtopia_Bot-Desktop_Qt_5_3_MSVC2013_32bit-Release\Makefile.Release [release\moc_mainwindow.obj] Error 1
'cl' is not recognized as an internal or external command,
operable program or batch file.
jom: C:\Users\mustafa\Desktop\Growtopia-Bot-master\build-Growtopia_Bot-Desktop_Qt_5_3_MSVC2013_32bit-Release\Makefile.Release [release\mainwindow.obj] Error 1
jom: C:\Users\mustafa\Desktop\Growtopia-Bot-master\build-Growtopia_Bot-Desktop_Qt_5_3_MSVC2013_32bit-Release\Makefile.Release [release\moc_addbots.obj] Error 1
jom: C:\Users\mustafa\Desktop\Growtopia-Bot-master\build-Growtopia_Bot-Desktop_Qt_5_3_MSVC2013_32bit-Release\Makefile [release] Error 2
22:43:06: The process "C:\Qt\Qt5.3.2\Tools\QtCreator\bin\jom.exe" exited with code 2.
Error while building/deploying project Growtopia_Bot (kit: Desktop Qt 5.3 MSVC2013 32bit)
When executing step "Make"
22:43:06: Elapsed time: 00:03.`

**Comment by Joiubaxas** [comment]
I've experimented a lot with getting the bot to even run without any success. I can compile the bot myself without any errors just a lot of warnings. When I start the program without a debugger i get this: The program has unexpectedly finished. When I attempt to do this with a cdb debugger I just get these lines:  

Debugging starts
Debugging has finished   

even if I set tons of breakpoints none of them trigger.
